views/js/pciCreator/dev/audioRecordingInteraction/runtime/js/recorder.js

```diff
@@ -134,8 +133,26 @@ define([
          */
         function getInputLevel() {
             var sum;
+            var frequencyLength;
+            var getRndInteger;
 
             analyser.getByteFrequencyData(frequencyArray);
+            frequencyLength = frequencyArray.length;
+
+            if (frequencyArray.length && /(iPhone|iPad)/i.test(navigator.userAgent)) {
+                getRndInteger = function(min, max) {
```

Comment:
Couldn't you put this function on a higher level? This looks like a helper, and this looks weird to have it define here.